include/LightGBM/config.h

```diff
@@ -260,7 +260,8 @@ struct Config {
   // desc = ``<= 0`` means disable
   int early_stopping_round = 0;
 
-  // desc = will only use first metric for early stopping if set to ``True``
+  // desc = set this to ``true``, if you want to use only the first metric for early stopping
+  // desc = **Note**: can be used only in CLI version
```

Comment:
Although this parameter is actually used in cli version, there is another parameter with the same name in python/R package. Therefore, I think the note is not needed, as well as the early_stopping_round.

Reply:
It's a little bit another situation (in Python, at least).
`early_stopping_round` is a parameter for `train`/`cv`, so everything is OK, but `first_metric_only` is a parameter for **callback**.
https://github.com/microsoft/LightGBM/blob/2a3691703c1f5aa5109881455cfc909f80c7f50f/python-package/lightgbm/callback.py#L153

Reply:
okay, I think the first_metric_only should be in train/cv as well, like the early_stopping_round:
https://github.com/microsoft/LightGBM/blob/2a3691703c1f5aa5109881455cfc909f80c7f50f/python-package/lightgbm/engine.py#L482-L483

Reply:
Do you think that we should try to retrieve `first_metric_only` from `params` and pass to callback?
https://github.com/microsoft/LightGBM/blob/2a3691703c1f5aa5109881455cfc909f80c7f50f/python-package/lightgbm/engine.py#L183-L184

Reply:
yes, just like the early stopping rounds:
https://github.com/microsoft/LightGBM/blob/2a3691703c1f5aa5109881455cfc909f80c7f50f/python-package/lightgbm/engine.py#L122-L126

Reply:
Got it! Then I'll create a PR for this and remove my note.